### DISCUSSION TITLE
linux/arm64/v8 not supported

### DISCUSSION THREAD

**Discussion by vantis-zh** [tiangolo/uvicorn-gunicorn-fastapi-docker#333] [Questions] [answered]
I'm trying to build an image on platform linux/arm64/v8, so I hope there could be an image available, thx.

**Comment by krzynio** [comment]
Hi, I need it as well ;)

**Comment by hemna** [comment]
same here.   I'm not sure why all of his docker images he builds don't support arm64 in general now.

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

If you build your Docker image from scratch, from the official Python image, you would have multi-arch (Arm, etc.) by default. :sunglasses:
